std/algebra/emulated/sw_bls12381/map_to_g1.go

```diff
@@ -6,13 +6,12 @@ import (
 	"github.com/consensys/gnark-crypto/ecc/bls12-381/fp"
 	"github.com/consensys/gnark-crypto/ecc/bls12-381/hash_to_curve"
 	"github.com/consensys/gnark/frontend"
-	"github.com/consensys/gnark/std/math/emulated"
 )
 
 func (g1 *G1) evalFixedPolynomial(monic bool, coefficients []fp.Element, x *baseEl) *baseEl {
 	emuCoefficients := make([]*baseEl, len(coefficients))
 	for i := range coefficients {
-		emulatedCoefficient := emulated.ValueOf[emulated.BLS12381Fp](coefficients[i])
+		emulatedCoefficient := *g1.curveF.NewElement(coefficients[i])
```

Comment:
does it make sense here to do "*value" and the next line to do "&result" ?

Reply:
It does not, indeed. Fixed